src/store/user/userReducer.js

```diff
@@ -3,7 +3,7 @@ import * as types from '../../contants/actionTypes'
 export default function user(state = {}, action) {
   switch (action.type) {
     case types.ADD_USER_TO_STATE: {
-      return { ...action.user }
+      return { ...state, ...action.user }
```

Comment:
Most probably this would never happen, but what about this scenario: 
First server responds with this user:
`user1 = { username: 'test', email: 'test@test.fi' }` 
And for some reason it is not cleared until server responds with
`user2 = { username: 'test2', someDataField: 'data' }`
What happens to email?

Reply:
Email would be linked with username test2? Removed ...state.